### ISSUE TITLE
change NODE_BIN detection

### ISSUE THREAD

**Issue by RoamingCookie** [extremeheat/JSPyBridge#99]
change
```python
NODE_BIN = getattr(os.environ, "NODE_BIN") if hasattr(os.environ, "NODE_BIN") else "node"
```
to 
```python
NODE_BIN = os.environ.get("NODE_BIN", "node")
```
at
https://github.com/extremeheat/JSPyBridge/blob/master/src/javascript/connection.py#L9

currently to change the node path we have to do this `setattr(os.environ, 'NODE_BIN', NODE_PATH)` then npm path like this `os.environ['NODE_PM'] = NPM_PATH` this is not interchangeable, but doing the recommend changes both could be change with the same (later) method

additionally allows the better use of python `nodejs-bin`  package to setup in remote python environments the current working method is

```python
import nodejs
import os
from pathlib import Path


setattr(os.environ, 'NODE_BIN', str(Path(nodejs.__file__).parent.joinpath("node.exe")))
os.environ['NODE_PM'] = str(Path(nodejs.__file__).parent.joinpath("npm.cmd"))

import javascript 

sass = javascript.require("sass")
```

```bash
pip install nodejs-bin javascript
```

**Comment by extremeheat** [comment]
You're right, this should be using .get(). Feel free to open a PR to fix. For compatibility in case anyone is depending on the attribute behavior instead of assigning to the dictionary, it would probably make sense to keep the current check as a fallback.
